### PR TITLE
fix: fix delete projects

### PR DESCRIPTION
**Pull request by Oluwadaminiola**

### CODE REVIEW

File: web/ui/dashboard/src/app/private/private.service.ts [hunk 1]
```diff
@@ -378,6 +378,21 @@ export class PrivateService {
 		});
 	}
 
+    deleteProject(): Promise<HTTP_RESPONSE> {
```
Comment (horlah) [inline web/ui/dashboard/src/app/private/private.service.ts:381]: @Oluwadaminiola Why did you move this here?

Comment (Oluwadaminiola) [reply web/ui/dashboard/src/app/private/private.service.ts:381]: I wanted to deprecate the settings service file as it only contained only one service @horlah

**Review by horlah — APPROVED** [review]
